.travis.yml

```diff
@@ -13,8 +13,11 @@
 # - basics - quick things like style and packaging
 # - test - the actual functional tests which are slow
 
+dist: trusty
 language: python
-cache: pip
+cache:
+  - pip
+  - apt: false
```

Comment:
Oh, I was tinkering the config file to see if not using any cache could update the build image to a recent version. I should have set the commit message with "[test]".